tests/Spec/Helpers.elm

```diff
@@ -1,9 +1,22 @@
 module Spec.Helpers exposing (..)
 
+import Expect exposing (Expectation)
 import Html.Attributes as Attributes
 import Test.Html.Selector as Selector
+import Test.Runner
 
 
 nriDescription : String -> Selector.Selector
 nriDescription desc =
     Selector.attribute (Attributes.attribute "data-nri-description" desc)
+
+
+expectFailure : String -> Expectation -> Expectation
```

Comment:
This is cool! Really neat way to test that keyboard events _don't_ happen! I'm excited about this